tools/web3-plugin-example/CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- EXAMPLE
+
+## [1.0.0]
+
+### Added
```

Comment:
Could this list the functionality that is added in this initial version?

Reply:
I believe that would be better suited for documentation, not a `CHANGELOG` - we also didn't do this for any other package

Reply:
It seems like this thing was missed early. So, if you would not add it in this MR, could you please ensure requesting to write this along with writing the documentation?  And then resolve this discussion point directly.
Thanks,,

Reply:
Updated via [this](https://github.com/web3/web3.js/pull/5393/commits/c939cbc428f00151ff392588b110d1cab12621a5) commit